habitat-lab/habitat/config/default_structured_configs.py

```diff
@@ -518,7 +521,7 @@ class BadCalledTerminateMeasurementConfig(MeasurementConfig):
 @dataclass
 class NavToPosSuccMeasurementConfig(MeasurementConfig):
     type: str = "NavToPosSucc"
-    success_distance: float = 0.2
+    success_distance: float = 1.5
```

Comment:
That is a pretty big increase in success threshold here. Is that okay?

Reply:
Yes, this is intentional since all skills are now mobile manipulation skills.